### PR TITLE
Updates access code text and relevant VR tests

### PR DESCRIPTION
**Pull request by balibirchlee**
### What is the context of this PR?

Updates text inline with style guidelines for the example access code pages.
https://github.com/ONSdigital/design-system/issues/2959

### How to review this PR

Check through code changes. Can also run VR tests. Perhaps check both of the access code pages in the components section locally.

### Checklist

This needs to be completed by the person raising the PR.

- [x] I have selected the correct Assignee
- [x] I have linked the correct Issue

### CODE REVIEW

File: src/components/access-code/example-access-code-error.njk [hunk 1]
```diff
@@ -58,28 +58,25 @@ layout: ~
...
+    <h2>If you do not have an access code</h2>
     <p>
-        You can <a href="#0">request a new access code</a> to start a new study if you have lost or not received an access code. This can be sent to you by text or post.
+        If you have lost or not received an access code, you can <a href="#0">request a new access code</a>.
+        This can be sent to you by text or post.
     </p>
 
     {% set content %}
-        <p>If you need to answer separately from the people you live with, you can <a href='#0'>request an individual access code</a> to start a separate study.</p>
+    <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```
Comment (rmccar) [inline src/components/access-code/example-access-code-error.njk:68]:
```suggestion
        <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```

File: src/components/access-code/example-access-code-error.njk [hunk 2]
```diff
@@ -58,28 +58,25 @@ layout: ~
...
     {% endset %}
 
     {% call onsDetails({
             "id": "details-access-code-error",
-            "title": "Need to answer separately from your household?",
-            "button": {
-                "close": "Hide this",
-                "contextSuffix": "content"
-            }
+            "title": "Need to answer separately from your household?"
```
Comment (rmccar) [inline src/components/access-code/example-access-code-error.njk:79]:
```suggestion
        "title": "Need to answer separately from your household?"
```
Id indenting needs fixing here too

File: src/components/access-code/example-access-code.njk [hunk 3]
```diff
@@ -34,28 +34,25 @@ layout: ~
...
+    <h2>If you do not have an access code</h2>
     <p>
-        You can <a href="#0">request a new access code</a> to start a new study if you have lost or not received an access code. This can be sent to you by text or post.
+        If you have lost or not received an access code, you can <a href="#0">request a new access code</a>.
+        This can be sent to you by text or post.
     </p>
 
     {% set content %}
-        <p>If you need to answer separately from the people you live with, you can <a href='#0'>request an individual access code</a> to start a separate study.</p>
+    <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```
Comment (rmccar) [inline src/components/access-code/example-access-code.njk:44]:
```suggestion
        <p>If you need to answer separately from the people you live with, you can <a href="#0">request an individual access code</a>.</p>
```

**Review by rmccar — COMMENTED** [review]
Just some code formatting

**Review by alessioventuriniAND — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Pull request merged**